app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt

```diff
@@ -58,6 +59,7 @@ class BaselinePingTest {
 
     private fun waitForPingContent(
         pingName: String,
+        pingReason: String?,
```

Comment:
Should we ... briefly document this function? for the sake of our future selves.